app/frontend/src/features/messages/Messages.tsx

```diff
@@ -44,7 +53,7 @@ export function HostRequestsSentNotification() {
 
 const labels = {
   all: "All",
-  groupchats: <MessagesNotification />,
+  chats: <MessagesNotification />,
```

Comment:
changed to chats for now since we aren't differentiating between group chats and dms in the ui